Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -302,6 +331,11 @@ namespace PhysX
     {
         m_debugDisplayDataChangeHandler.Disconnect();
 
+        if (m_sceneConfigChangedHandler.IsConnected())
+        {
+            m_sceneConfigChangedHandler.Disconnect();
```

Comment:
Not sure how it works with events, but I think with buses you can safely call Disconnect even if you're not connected, so you don't need to wrap it in a check to see if it's connected. Maybe worth checking if this is necessary for events.

Reply:
That's correct, the `IsConnected` check happens already in `Disconnect` itself